### ISSUE TITLE
explicitly state the challenges that this project intends to address

### ISSUE THREAD

**Issue by christopher-bennage** [mspnp/data-pipeline#14]
It was pointed out (thanks @kellabyte) that we are not describing the problems that this code is intended to address. Let's add them to the root readme.

**Comment by kellabyte** [comment]
Yeah, without a problem, some of the stated solutions may or may not be required. Things like persisted storage or what kind of consistency guarantees you need.
